crc/scripts/set_study_status.py

```diff
@@ -0,0 +1,29 @@
+from crc import session
+from crc.api.common import ApiError
+from crc.models.study import StudyModel, StudyStatus
+from crc.scripts.script import Script
+
+
+class SetStudyStatus(Script):
+
+    def get_description(self):
+        return """Set the study status. Requires a study status.
+        Study status must be 'in_progress', 'hold', 'open_for_enrollment', 'abandoned', or 'cr_connect_complete'."""
+
+    def do_task_validate_only(self, task, study_id, workflow_id, *args, **kwargs):
+        if len(args) == 1:
+            if args[0] in ['in_progress', 'hold', 'open_for_enrollment', 'abandoned', 'cr_connect_complete']:
```

Comment:
Keep it DRY - would be better to use the enum here, otherwise we have to update the enum, and three places in this file if we change the study statuses.